### PR TITLE
Spec clean up

### PR DESCRIPTION
**Pull request by codyj110**

### CODE REVIEW

**Comment by codyj110** [comment]
not sure why appveyor is failing my node and  yarn versions match but i don't receive error when running yarn test:win32

**Comment by jondot** [comment]
Amazing! I love that you contributed for tests, often times they need more love from contributors.
May I ask what made you feel like the tests need clean up?

**Comment by codyj110** [comment]
Since i am using your tool i wanted to help out so to further understand your code base i started to look at the unit test and found they were hard to follow at first glance so i figured that would be a great place to start contributing to this project.

**Comment by jondot** [comment]
Great, really happy you took the time to work on tests!
Merging!
